tuna/dbBase/base_class.py

```diff
@@ -25,17 +25,21 @@
 #
 ###############################################################################
 """ Module for creating DB tables interfaces"""
+from typing import Dict, Any, List
 from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.ext.declarative.api import DeclarativeMeta
 from sqlalchemy.sql import func as sqla_func
 from sqlalchemy.dialects.mysql import TINYINT
 from sqlalchemy import Column, Integer, DateTime, text
+from sqlalchemy.schema import Table
 
 
 class BASE():
   """Base class for our own common functionalities among tables"""
 
-  __table_args__ = {'mysql_engine': 'InnoDB'}
-  __mapper_args__ = {'always_refresh': True}
+  __table__: Table = None
+  __table_args__: Dict[str, str] = {'mysql_engine': 'InnoDB'}
```

Comment:
`Dict[str, All]` would be more general, covering non-string type arguments as well